# duration
float32 duration

# trajectory sequence
int32 sequence

# start time
float64 start_time

# type
string type

# sampling time
float32 h

# trajectory in x
float32[] traj_x

# trajectory in y
float32[] traj_y

float32[] traj_theta